packages/experimental/Dropdown/src/Option/Option.styling.ts

```diff
@@ -2,21 +2,25 @@ import { borderStyles, buildProps, fontStyles, layoutStyles, Theme, UseStylingOp
 import { optionName, OptionProps, OptionSlotProps, OptionTokens } from './Option.types';
 import { defaultOptionTokens } from './OptionTokens';
 
+export const optionStates: (keyof OptionTokens)[] = ['hovered', 'focused', 'pressed', 'disabled'];
```

Comment:
It feels like we might or should have a type for these 4 states in particular we can spread? Future fodder if not.. I don't know that would make code cleaner..

Reply:
I don't know how well that would play with typing. We can try it, I guess a template might make it feasible

Reply:
I think I was confusing old and new framework, and different types. `useAsPressable.types.ts` has "I<Focus | Press | Hover>State` types. Nevermind..